gumbo-millennium/website
# Simplify image handling by using native filesystem

## What's changed

- [x] Added Image fluent for easy generation of URLs
- [ ] Replaced use of paperclip with regular images
- [ ] Replace inline use of paperclip sizes with image fluent
- [ ] Remove paperclip package
- [ ] Add migration command
- [ ] Locally store Shop images and scale them accordingly

## Testing

- Check all images for current and past images still work.
- Check migration works properly
